src/TransTableL.cpp

```diff
@@ -1025,8 +1025,8 @@ void TransTableL::PrintNodeValues(
     cardSuit[3] << cardRank[15-static_cast<int>(np.leastWin[3])] << "\n";
 
   fout << setw(16) << left << "Bounds" << 
-    to_string(np.lbound) << " to " <<
-    to_string(np.ubound) << " tricks\n";
+    to_string(static_cast<int>(np.lbound)) << " to " <<
```

Comment:
Why is unsigned char generate warning when calling to_string?

Reply:
I was stupid. Explanation was in the commit message. The github provided horrible formating didn't do any favors for reading the compiler warning.